tests/integration/cases/micronaut-projects_micronaut-core/config.ini

```diff
@@ -7,7 +7,7 @@ exclude =
     # temporarily because provenances have failed to publish due to an issue in `generator_generic_slsa3.yml@v1.9.0`:
     # https://github.com/slsa-framework/slsa-github-generator/issues/3350
     mcn_provenance_available_1
-    # Exclude `mcn_infer_artifact_pipeline_1`, due to a non-deterministic behavior in deploy command detection,
+    # Exclude `cn_find_artifact_pipeline_1`, due to a non-deterministic behavior in deploy command detection,
     # which will be fixed in PR #673.
-    mcn_infer_artifact_pipeline_1
+    cn_find_artifact_pipeline_1
```

Comment:
What happened here?

Reply:
Oops! We actually don't need to exclude this as the issue has been addressed. See commit [c9761dc](https://github.com/oracle/macaron/pull/872/commits/c9761dca9c853e65a196c033ebe45b53bbf3ac05).